collect_app/src/main/java/org/odk/collect/android/utilities/VersionHidingCursorAdapter.java

```diff
@@ -50,12 +50,23 @@ public boolean setViewValue(View view, Cursor cursor,
                 } else {
                     String version = cursor.getString(columnIndex);
                     TextView v = (TextView) view;
+                    v.setText("");
+                    v.setVisibility(View.GONE);
                     if (version != null) {
-                        v.setText(String.format(ctxt.getString(R.string.version_number), version));
+                        v.append(String.format(ctxt.getString(R.string.version_number), version));
                         v.setVisibility(View.VISIBLE);
                     } else {
-                        v.setText(null);
-                        v.setVisibility(View.GONE);
+                        v.append("");
```

Comment:
Is this `else` necessary? Appending `""` should have no effect.

Reply:
Yes @lognaturel, I also thought the same but since it was there in the original code so i just included it in my code too. Thanks for pointing it out. I'll fix it.